src/main/scala/com/typesafe/sbt/license/LicenseCategory.scala

```diff
@@ -29,6 +29,7 @@ object LicenseCategory {
   val PublicDomain = LicenseCategory("Public Domain")
   val CC0 = LicenseCategory("CC0", Seq("Creative Commons Zero"))
   val NoneSpecified = LicenseCategory("none specified")
+  val Unrecognized = LicenseCategory("")
```

Comment:
Maybe this should be "unrecognized" as a string....  Or something that's not empty.